catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -8,26 +8,32 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
 
 We will keep this page up to date with the latest information as we make more progress.
 
-## What’s new? (updated 8 July 2020)
+## What’s new? (updated 14 October 2020)
 
-### Improvements to search relevance
-
-#### Fine tuning relevance
-Based on the data we already have about how search is being used, we've tested a change to the way queries are split and combined. Rather than letting results with _any_ (one or more) of the terms appear in your results (using the OR operator between query tokens), we now require that _all_ of the tokens appear in any result you see (using the AND operator between query tokens). This seems to be returning more relevant results, so we've made this the default algorithm, while kicking off a new test.
+### Improvements to search
+- Up to now we have only exposed a subset of digital and digitised materials in search results, you will now also be able to find archives and physical works from our collections.
```

Comment:
I wonder if we could follow a structure similar to [what's mentioned here](https://uxdesign.cc/the-art-of-writing-great-release-notes-6607e22efae1).

* What
* Why
* User impact

e.g.

#### Full catalogue search
We were previously exposing only a digital and digitised materials subset of our catalogue in search. We now include our whole catalogue, including archives, physical works from our collections, and more.

This will mean you will get a lot more results back for a given search as we are now searching approximately 1.2 millions results, as opposed to the +-300,000 results we previously searched.

To help you find more of what you need, our current filters should help, and we have added a new search filter.

### Online / In library search filter
...